src/EditorFeatures/CSharpTest/ConvertAutoPropertyToFullProperty/ConvertAutoPropertyToFullPropertyTests.cs

```diff
@@ -0,0 +1,985 @@
+    private int _goo = 2 * num;
+
+    public int Goo
+    {
+        get
+        {
+            return _goo;
+        }
+        set
+        {
+            _goo = value;
+        }
+    }
+}
+";
+            await TestInRegularAndScriptAsync(text, expected, options: DoNotPreferExpressionBodiedAccessors);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.ConvertAutoPropertyToFullProperty)]
+        public async Task WithPrivateSetter()
+        {
+            var text = @"
+class goo
+{
+    public int G[||]oo { get; private set; }
+}
+";
+            var expected = @"
+class goo
+{
+    private int _goo;
+
+    public int Goo
+    {
+        get
+        {
+            return _goo;
+        }
+        private set
+        {
+            _goo = value;
+        }
+    }
+}
+";
+            await TestInRegularAndScriptAsync(text, expected, options: DoNotPreferExpressionBodiedAccessors);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.ConvertAutoPropertyToFullProperty)]
+        public async Task WithFieldNameAlreadyUsed()
+        {
+            var text = @"
+class goo
+{
+    private int _goo;
+
+    public int G[||]oo { get; private set; }
+}
+";
+        var expected = @"
+class goo
+{
+    private int _goo;
+    private int _goo1;
+
+    public int Goo
+    {
+        get
+        {
+            return _goo1;
+        }
+        private set
+        {
+            _goo1 = value;
+        }
+    }
+}
+";
+            await TestInRegularAndScriptAsync(text, expected, options: DoNotPreferExpressionBodiedAccessors);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.ConvertAutoPropertyToFullProperty)]
+        public async Task WithComments()
+        {
+            var text = @"
+class goo
+{
+    // Comments before
+    public int G[||]oo { get; private set; } //Comments during
+    //Comments after
+}
+";
+        var expected = @"
+class goo
+{
+    private int _goo;
+
+    // Comments before
+    public int Goo
+    {
+        get
+        {
+            return _goo;
+        }
+        private set
+        {
+            _goo = value;
+        }
+    } //Comments during
+    //Comments after
+}
+";
+            await TestInRegularAndScriptAsync(text, expected, options:DoNotPreferExpressionBodiedAccessors);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.ConvertAutoPropertyToFullProperty)]
+        public async Task WithExpressionBody()
+        {
+            var text = @"
+class goo
+{
+    public int G[||]oo { get; set; }
+}
+";
+        var expected = @"
+class goo
+{
+    private int _goo;
+
+    public int Goo { get => _goo; set => _goo = value; }
+}
+";
+            await TestInRegularAndScriptAsync(text, expected, options: PreferExpressionBodiedAccessorsWhenPossible);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.ConvertAutoPropertyToFullProperty)]
+        public async Task WithExpressionBodyWhenOnSingleLine()
+        {
```

Comment:
Is this input supposed to be the same as the previous test? Should there be a test for PreferExpressioNBodiedAccessorsWhenOnSingleLIne with the original Goo definition spanning multiple lines?

Reply:
Added this test